### PR TITLE
Throw a seal in the finance channel

### PR DESCRIPTION
**Pull request by pablolee**

### CODE REVIEW

**Comment by stephnacios** [comment]
:+1: LGTM!

**Comment by jiexinhuang** [comment]
:+1: :tada: :fish:

File: config/envato.yml [hunk 1]
```diff
@@ -63,3 +63,15 @@ site-reliability:
   use_labels: true
   exclude_labels:
     - wip
+
+market-financedev:
+  channel: '#mp-financedev'
```
Comment (orien) [inline config/envato.yml]: here or `#mp-financedev-prs`?

**Comment by orien** [comment]
+1

File: config/envato.yml (hunk 1 above)
Comment (pablolee) [inline config/envato.yml]: Oh, good point. `#mp-financedev-prs` is more appropriate but `#mp-financedev` is more "in your face". Let's start out in  `#mp-financedev-prs` and escalate from there

**Comment by johnsyweb** [comment]
:fish:

**Comment by andredantasrocha** [comment]
What is the frequency of the alerts? do we need something different for our PRs?

**Comment by johnsyweb** [comment]
Seal uses the [Heroku scheduler addon](https://addons-sso.heroku.com/apps/envato-seal/addons/e30a77c3-e871-4e5b-8ef0-0fde15431268) to run at 22:00 UTC. It can be invoked on-demand, if you prefer.

**Comment by pablolee** [comment]
Now uses `github_team_id` to specify people

**Pull request merged**